src/user/using-the-aggregate-data-entry-app.md

```diff
@@ -0,0 +1,190 @@
+
+### Entering data { #entering-data-in-a-data-entry-form }
+
+Once you've opened a form, you can start entering data into the form cells. The active cell, the cell that you're entering data into, is always highlighted with a blue border. To work quickly, you can move around using your keyboard:
+- to move on to the next cell, press `tab` or `↓`.
+- to go back to the previous cell, press `shift + tab` or `↑`.
+
+#### Cell status { #data-entry-form-cell-status }
+Cells look different depending on what their status is:
+
+| Cell    | Status                                                                                                                                           |
+| ------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
+| ![](resources/images/data_entry_beta/cell-normal.png)  | Showing a value that's already saved or a cell that's empty.                                                                                            |
+| ![](resources/images/data_entry_beta/cell-synced.png)   | The cell value is saved on the server.                                                                                                                |
+| ![](resources/images/data_entry_beta/cell-waiting.png) | The cell value is saved locally and is syncing, or waiting to sync, to the server.                                                                                              |
+| ![](resources/images/data_entry_beta/cell-invalid.png)     | There's a problem with the cell value. Click or hover over the cell to learn more about the problem. These invalid values are not saved to the server or locally. |
+| ![](resources/images/data_entry_beta/cell-comment.png) | The value has a comment.                                                                                                                         |
+| ![](resources/images/data_entry_beta/cell-readonly.png)  | The cell is locked and the value can't be edited.                                                                                                                                                 |
+
+### Filtering a form { #filtering-data-entry-form }
+Filtering is useful if you're looking for a certain cell in a form. You can filter the whole form, filter single sections, or both. Any cells that don't match the filter will be hidden.
+
+#### Filtering the whole form form { #filtering-whole-data-entry-form }
+
+![](resources/images/data_entry_beta/filter-form.png) { width=60% }
+
+To filter the whole data entry form, enter a value into the input at the top of the form.
+
+#### Filtering a section { #filtering-section-data-entry-form }
+
+![](resources/images/data_entry_beta/filter-section.png) { width=60% }
+
+For section form you can filter only inside a single section. Enter a value into the input at the top of a section.
```

Comment:
Sounds a little awkward/ambiguous to me. Perhaps:
```suggestion
For section forms you can also filter inside a single section. Enter a value into the input at the top of a section.
```

Reply:
Thanks, committed.